internal/config.go

```diff
@@ -64,15 +65,42 @@ func LoadConfig(path string) (*Config, error) {
 	return &c, nil
 }
 
-func (c *Config) fillAndValidate() error {
-	if c.Results.Common == nil {
-		if c.Results.FerretDB == nil || c.Results.MongoDB == nil {
-			return fmt.Errorf("both FerretDB and MongoDB results must be set (if common results are not set)")
+// Merge common config with both databases.
```

Comment:
@noisersup  well, here if a linter for unexported funcs will be enabled, it will argue that the comment doesn't start with the func name. 
```suggestion
// mergeTestConfigs merges common config with both databases test configs.
```